cmd/image/import/import.go

```diff
@@ -178,8 +178,8 @@ pvsadm image import -n upstream-core-lon04 -b <BUCKETNAME> --object-name rhel-83
 }
 
 func init() {
-	Cmd.Flags().StringVarP(&pkg.ImageCMDOptions.InstanceName, "instance-name", "n", "", "Instance name of the PowerVS")
-	Cmd.Flags().StringVarP(&pkg.ImageCMDOptions.InstanceID, "instance-id", "i", "", "Instance ID of the PowerVS instance")
+	Cmd.Flags().StringVarP(&pkg.ImageCMDOptions.InstanceName, "powervs-instance-name", "n", "", "Name of the PowerVS instance")
+	Cmd.Flags().StringVarP(&pkg.ImageCMDOptions.InstanceID, "powervs-instance-id", "i", "", "ID of the PowerVS instance")
```

Comment:
should we use powervs or pvs as the prefix? 
I would strongly suggest to have a consistent naming of options so that it's easy for users to remember
for example pvs-instance-name, pvs-instance-id, pvs-service-cred etc...